packages/js/tests/dashboard/components/page-title.test.js

```diff
@@ -0,0 +1,83 @@
+import { describe, expect, it } from "@jest/globals";
+import { PageTitle } from "../../../src/dashboard/components/page-title";
+import { render } from "../../test-utils";
+
+/**
+ * @param {Container} container The container.
+ * @param {Function} getByRole The getByRole function.
+ * @returns {void}
+ */
+const verifyDefaultContent = ( container, getByRole ) => {
+	expect( getByRole( "heading", { name: "Hi John," } ) ).toBeInTheDocument();
+
+	const text = {
+		description: "Welcome to your dashboard! Check your content's SEO performance, readability, and overall strengths and opportunities.",
+		contentAnalysisLink: "Learn more on how to improve your content with our content analysis tool",
+		contentAnalysisLinkScreenReader: "(Opens in a new browser tab)",
+	};
+
+	// The text is not retrievable with getByText due to the HTML. So below is a bit strange with the joins with and without spaces.
+	expect( container.querySelector( "p" ).textContent )
+		.toBe( [ [ text.description, text.contentAnalysisLink ].join( " " ), text.contentAnalysisLinkScreenReader, "." ].join( "" ) );
+
+	const contentAnalysisLink = getByRole( "link", {
+		name: [ text.contentAnalysisLink, text.contentAnalysisLinkScreenReader ].join( " " ),
+	} );
+	expect( contentAnalysisLink ).toHaveAttribute( "href", "https://example.com/content-analysis" );
+	expect( contentAnalysisLink ).toHaveAttribute( "target", "_blank" );
+	expect( contentAnalysisLink ).toHaveTextContent( text.contentAnalysisLink );
+};
+
+describe( "PageTitle", () => {
+	it( "should render the component", () => {
+		const { container, getByRole } = render(
+			<PageTitle
+				userName="John"
+				features={ { indexables: true, seoAnalysis: true, readabilityAnalysis: true } }
+				links={ { contentAnalysis: "https://example.com/content-analysis" } }
+			/>
+		);
+
+		verifyDefaultContent( container, getByRole );
+	} );
+
+	it( "should show a warning with SEO and readability both disabled", () => {
+		const { container, getByRole } = render(
+			<PageTitle
+				userName="Bob"
+				features={ { indexables: true, seoAnalysis: false, readabilityAnalysis: false } }
+				links={ { contentAnalysis: "https://example.com/content-analysis" } }
+			/>
+		);
+
+		expect( getByRole( "heading", { name: "Hi Bob," } ) ).toBeInTheDocument();
+
+		// The text is not retrievable with getByText due to the HTML.
+		expect( container.querySelector( "p" ).textContent )
+			.toBe( "It looks like the ‘SEO analysis’ and the ‘Readability analysis’ are currently disabled in your Site features or your user profile settings. Enable these features to start seeing all the insights you need right here!" );
+
+		const siteFeaturesLink = getByRole( "link", { name: "Site features" } );
+		expect( siteFeaturesLink ).toHaveAttribute( "href", "admin.php?page=wpseo_page_settings#/site-features" );
```

Comment:
If there is a plan to move that code to a separate package, shouldn't we make it as much platform dependent as possible? By having those WordPress related links as props.

Reply:
No plan. But these messages were deemed temporary due to there "only" being 2 widgets and no widget control yet.